Luke_Smith_Portfolio.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Validator's complaining that you haven't declared a language variable. Adding "lang=en-GB" should keep it happy

Reply:
The html tag needs to be given the attribute lang. Since you've written the page in British English, add "lang=en-GB" to make this valid html.